servers/audio/effects/eq.cpp

```diff
@@ -73,9 +74,9 @@ void EQ::recalculate_band_coefficients() {
 
 
 
-		double side_gain2=POW2(1.0/M_SQRT2);
-		double th=2.0*M_PI*frq/mix_rate;
-		double th_l=2.0*M_PI*frq_l/mix_rate;
+		double side_gain2=POW2(1.0/Math_SQRT2);
```

Comment:
Lol :) Just do `double side_gain2=POW2(Math_SQRT12);` and that's one less constant to add in math_funcs :)